js/stores/windowStore.js

```diff
@@ -841,7 +813,7 @@ const doAction = (action) => {
       if (getSetting(settings.AUTO_HIDE_MENU)) {
         doAction({actionType: WindowConstants.WINDOW_TOGGLE_MENUBAR_VISIBLE, isVisible: false})
       } else {
-        hideContextMenu(action)
+        doAction({actionType: WindowConstants.WINDOW_SET_CONTEXT_MENU_DETAIL})
       }
       doAction({actionType: WindowConstants.WINDOW_SET_SUBMENU_SELECTED_INDEX})
       doAction({actionType: WindowConstants.WINDOW_SET_BOOKMARKS_TOOLBAR_SELECTED_FOLDER_ID})
```

Comment:
Glad this came up- this is something I missed while doing the refactor. Resetting the menu status is supposed to clear the selected folder ID. This action never got renamed to the new one